webpack/components/PreupgradeReportsList/components/PreupgradeReportEntry.js

```diff
@@ -37,14 +37,7 @@ const PreupgradeReportEntry = ({ entry, isEntrySelected, toggleSelected }) => (
       <ListView.InfoItem key={itemIteratorId(entry, entry.hostname)}>
         <Icon type="pf" name="cluster" /> {entry.hostname}
       </ListView.InfoItem>,
-      <ListView.InfoItem key={itemIteratorId(entry, entry.severity)}>
-        <Icon
-          type="pf"
-          name="resources-full"
-          className={severityToCssClass(entry.severity)}
-        />
-        {entry.severity}
-      </ListView.InfoItem>,
+      getSeverityImg(entry),
```

Comment:
I believe this has to be wrapped in `ListView.InfoItem`, otherwise the items don't show up aligned in columns

Reply:
I thought I put into that... sorry.. fixing..